src/screens/settings/css/notification.css

```diff
@@ -23,4 +23,105 @@
     src: url(../fonts/Rubik-VariableFont_wght.ttf);
   }
 
-  /* DO NOT REMOVE THE CODE ABOVE */
\ No newline at end of file
+  /* DO NOT REMOVE THE CODE ABOVE */
+
+  /* /-----------------------SIDE BAR -------------------------/ */
+:root {
+  /* General */
+  --svg-color: #a9a9a9;
+  /* light mode */
+  --lm-sidebar-bg-color: #22074a;
+  --lm-active-menu-bg-color: #ffffff;
+  --lm-active-menu: rgba(34, 32, 32, 0.9);
+
+  /* dark mode */
+  --dm-sidebar-bg-color: #1f201f;
+  --dm-active-menu: #fffcfc;
+  --dm-active-menu-bg-color: #373737;
+}
+
+/*/------------------- CSS FOR SIDEBAR LIGHT MODE -------------------/*/
+
+.light-sidebar-nav,
+.dark-sidebar-nav {
+  width: 310px;
+  padding: 43px 0 41px 0;
+  display: flex;
+  flex-direction: column;
+  gap: 41px;
+}
+
+.light-sidebar-nav {
+  background-color: var(--lm-sidebar-bg-color);
+}
+
+.dark-sidebar-nav {
+  background-color: var(--dm-sidebar-bg-color);
+  background-image: url("../../../assets/icons/worldwide-location.svg"),
+    url("../../../assets/icons/chat-room.svg"),
+    url("../../../assets/icons/worldwide-location.svg"),
+    url("../../../assets/icons/video-conference.svg"),
+    url("../../../assets/icons/chat-room.svg"),
+    url("../../../assets/icons/video-conference.svg");
+  background-repeat: no-repeat, no-repeat, no-repeat, no-repeat, no-repeat, no-repeat;
+  background-position: top 56px right 92px, top 198px right 18px, top 395px right 250px,
+    top 450px right 18px, top 547px right 160px, top 789px right 231px;
+}
+
+header .light-prof-pic,
+.light-menu-link {
+  padding-left: 72px;
+}
+
+.light-nav-link-container,
```

Comment:
since there are of the same properties, u can simply just use on class, like say nav-link-container